web_src/less/_base.less

```diff
@@ -1569,6 +1575,10 @@ a.ui.label:hover {
   }
 }
 
+.lines-escape {
+  width: 0;
+}
```

Comment:
Why do we need this? It seems there is content in `<td class="lines-escape">`

Reply:
Width width 0:

![Screenshot from 2022-01-06 19-40-48](https://user-images.githubusercontent.com/1824502/148444247-6d7f79b4-ca03-467b-a65b-de91478895d6.png)

Without width 0:

![Screenshot from 2022-01-06 19-40-55](https://user-images.githubusercontent.com/1824502/148444274-eea8308e-d2b4-4b34-8a2a-3c222d9735bd.png)